client/controllers/incidentReports.coffee

```diff
@@ -27,6 +27,18 @@ Template.incidentReports.helpers
         label: "Reported"
         fn: (value, object, key) ->
           return moment(value).fromNow()
+      },
+      {
+        key: "travelRelated"
+        label: "Travel Related"
+        fn: (value, object, key) ->
+          if value
+            return "Yes"
+          return "No"
```

Comment:
When the travel related box isn't checked, we should just leave fields like this blank instead of interpreting it as meaning not travel related. The check box could be empty because the travel status is unknown or because it is an old record from before the field was added.